### PR TITLE
Move Android dependencies from support library to androidx

### PR DESCRIPTION
**Pull request by SerhiiBovtriuk**
Currently blocked because of issues with Robolectric.

### CODE REVIEW

File: connect/build.gradle [hunk 1]
```diff
@@ -45,6 +45,7 @@ dependencies {
     api 'com.squareup.okhttp3:logging-interceptor:3.11.0'
     api 'com.squareup.retrofit2:retrofit:2.4.0'
     api 'com.squareup.retrofit2:converter-gson:2.4.0'
+    implementation 'com.google.android.material:material:1.0.0'
```
Comment (jorunfa) [inline connect/build.gradle]: What are we using this dependency for? Also, is it on purpose that it's `implementation` and not `api`?

File: connect/tests/build.gradle [hunk 2]
```diff
@@ -20,13 +21,14 @@ android {
...
-        testImplementation ("org.robolectric:robolectric:3.4.2") {
+        testImplementation ("org.robolectric:robolectric:4.2") {
             exclude module: 'bcprov-jdk16'
         }
-        testImplementation ('org.robolectric:shadows-support-v4:3.0') {
+        testImplementation ("org.robolectric:shadows-support-v4:3.0") {
             exclude module: 'bcprov-jdk16'
         }
         testImplementation 'junit:junit:4.12'
+        testImplementation 'com.google.android.material:material:1.0.0'
```
Comment (jorunfa) [inline connect/tests/build.gradle]: Same question as above.

**Comment by jorunfa** [comment]
I'm getting `WARNING: The option setting 'android.enableUnitTestBinaryResources=true' is experimental and unsupported.`
I'm guessing we're using/need this?

**Comment by SerhiiBovtriuk** [comment]
This is moved to another pull request. Keep track on https://github.com/telenordigital/connect-android-sdk/pull/179

**Pull request closed without merging**